web-services-sample-app/GetWeatherVC.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ViewController.swift
+//  web-services-sample-app
+//
+//  Created by webonise on 29/09/16.
+//  Copyright © 2016 webonise. All rights reserved.
+//
+
+import UIKit
+
+class GetWeatherVC: UIViewController {
+    
+    @IBOutlet var textFieldEnterCityName: UITextField!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+    }
+
+    /** This function checks for validations whether the user has added only whitespaces in the text field
```

Comment:
The format for documentation is not right, refer the previous comments for the right format